### DISCUSSION TITLE
Does cross-seed also match music and other categories?

### DISCUSSION THREAD

**Discussion by TitanicKiwi** [cross-seed/cross-seed#403] [General]
It is unclear to me what categories of torrents cross-seed can match. 
Can it just match TV series and movies, or can it do other categories? Music? Books? Games? Software?

Also, is there an explanation of how the matching algorithm works? Does it match by the size of the torrent content, and the name of the torrent (or of the top-level folder in the torrent or the name of the single file, if it's just one)?
If it can match series, why wouldn't it also be able to match Music?

**Comment by tordenflesk** [comment]
enable includeNonVideos
